vsintegration/src/FSharp.Editor/Navigation/GoToDefinition.fs

```diff
@@ -199,6 +199,76 @@ type internal GoToDefinition(metadataAsSource: FSharpMetadataAsSourceService) =
                 return None
         }
 
+    member _.TryGetExternalDeclarationAsync(targetSymbolUse: FSharpSymbolUse, metadataReferences: seq<MetadataReference>) =
```

Comment:
Tiny nit: this could probably be a `let`-binding here instead of a public `member`.

Reply:
Yes I tried IIRC, but it's in use outside. This whole thing should really be modules, it doesn't hold state and is not MEF composed, but refactoring it all is a rabbit-hole.